packages/components/src/link/src/Link.css

```diff
@@ -157,19 +165,25 @@
 }
 
 /* ICON */
-.o-ui-link-start-icon,
-.o-ui-link-end-icon {
+.o-ui-text-link .o-ui-text-link-start-icon,
+.o-ui-text-link .o-ui-text-link-end-icon {
+    width: 1em;
```

Comment:
shouldn't we use --o-ui-sz-2 here?

Reply:
No since rem's are based on the root font size while em's are based on the parent font size. That would make all icons the same width.

Reply:
my bad did not notice